sdk-extensions/metric-incubator/src/test/java/io/opentelemetry/sdk/viewconfig/ViewConfigCustomizerTest.java

```diff
@@ -0,0 +1,130 @@
+              callback.record(
+                  1,
+                  Attributes.builder()
+                      .put("foo", "val")
+                      .put("bar", "val")
+                      .put("baz", "val")
+                      .put("qux", "val")
+                      .build());
+            });
+
+    assertThat(reader.collectAllMetrics())
+        .hasSize(1)
+        .satisfiesExactly(
+            metricData -> {
+              assertThat(metricData)
+                  .hasLongSum()
+                  .points()
+                  .satisfiesExactly(
+                      point ->
+                          assertThat(point)
+                              .hasValue(1)
+                              .hasAttributes(
+                                  Attributes.builder()
+                                      .put("foo", "val")
+                                      .put("bar", "val")
+                                      .build()));
+            });
+  }
+
+  @Test
+  void customizeMeterProvider_MultipleFiles() {
+    ConfigProperties properties =
+        withConfigFileLocations(
+            Arrays.asList(
+                ViewConfigTest.class.getResource("/view-config-customizer-test.yaml").getFile(),
+                ViewConfigTest.class.getResource("/full-config.yaml").getFile()));
+
+    assertThatCode(
+            () ->
+                ViewConfigCustomizer.customizeMeterProvider(SdkMeterProvider.builder(), properties))
+        .doesNotThrowAnyException();
+  }
+
+  @Test
+  void customizeMeterProvider_ClassPathFile() {
+    ConfigProperties properties =
+        withConfigFileLocations(
+            Collections.singletonList("classpath:/view-config-customizer-test.yaml"));
+
+    assertThatCode(
+            () ->
+                ViewConfigCustomizer.customizeMeterProvider(SdkMeterProvider.builder(), properties))
+        .doesNotThrowAnyException();
+  }
+
+  @Test
+  void customizeMeterProvider_Invalid() {
+    assertThatThrownBy(
+            () ->
+                ViewConfigCustomizer.customizeMeterProvider(
+                    SdkMeterProvider.builder(),
+                    withConfigFileLocations(
+                        Collections.singletonList("classpath:" + UUID.randomUUID()))))
+        .hasMessageContaining("Resource not found on classpath:");
+    assertThatThrownBy(
+            () ->
+                ViewConfigCustomizer.customizeMeterProvider(
+                    SdkMeterProvider.builder(),
+                    withConfigFileLocations(
+                        Collections.singletonList(
+                            ViewConfigTest.class
+                                .getResource("/empty-selector-config.yaml")
+                                .getFile()))))
+        .hasMessageContaining("Failed to parse view config")
+        .hasRootCauseMessage("selector is required");
+  }
+
+  private static ConfigProperties withConfigFileLocations(List<String> fileLocations) {
```

Comment:
Tiny nit you could make this varargs and have the `Arrays.asList` in here instead